README.md

```diff
@@ -747,8 +747,8 @@ curl -X GET \
 
 #### Response:
 
-On Start up the context held nine products, three more have been added by the
-create operations so the full context will return twelve products.
+On start up the context held nine products, three more have been added by
```

Comment:
Yes the capital was incorrect, but actually **start-up** would be  grammatically correct. (Chicago Manual of Style again)

**start up** is the verb, **start-up** is the noun and phase of operation

https://www.quora.com/Which-is-correct-startup-start-up-or-start-up

Reply:
Excellent, thanks!!!